### PR TITLE
add filesdb recipe

### PR DESCRIPTION
**Pull request by stilley2**
Pull request to add filesdb https://pypi.python.org/pypi/filesdb to conda-forge

### CODE REVIEW

File: recipes/filesdb/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  number: 0
+  entry_points:
+    - filesdb = filesdb.__init__:main
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python >=3
```
Comment (bgruening) [inline recipes/filesdb/meta.yaml]: can you skip the python3 buidls in the build section please `build/skip: True  # [py2k]`

File: recipes/filesdb/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  requires:
+    - pytest
+
+about:
+  home: https://github.com/stilley2/filesdb
+  license: MIT
+  license_family: MIT
+  license_file: 'LICENSE.txt'
+  summary: A simple tool for tracking files
+  description: 'UNKNOWN
```
Comment (bgruening) [inline recipes/filesdb/meta.yaml]: please add a proper description

File: recipes/filesdb/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+
+
+    '
+  doc_url: 'https://github.com/stilley2/filesdb'
+  dev_url: 'https://github.com/stilley2/filesdb'
+
+extra:
+  recipe-maintainers: 'Steven Tilley'
```
Comment (bgruening) [inline recipes/filesdb/meta.yaml]: please use the list syntax here:
```
recipe-maintainers: 
  - @github_username
```

**Comment by synapticarbors** [comment]
@stilley2 -- Thanks for your contribution to conda-forge. A feedstock for this recipe should be generated soon and you'll be receiving several automated messages about it. When the feedstock has been rendered, please take a look at the README in that repo for information about updating and maintaining the recipe. One of the key points is all updates should be done by forking that repo and following a standard pull request workflow. If you haven't looked at it already the docs are quite helpful:

https://conda-forge.org/docs/

If you run into any difficulties please let us know.

**Pull request merged**